agr_literature_service/lit_processing/data_ingest/dqm_ingest/sort_dqm_json_reference_updates.py

```diff
@@ -277,6 +277,16 @@ def sort_dqm_references(input_path, input_mod, base_dir=base_path):      # noqa:
     missing_papers_in_mod = {}
     missing_agr_in_mod = {}
 
+    search_path = path.dirname(path.abspath(__file__)).replace("/dqm_ingest", "") + \
+        "/pubmed_ingest/data_for_pubmed_processing/"
+    exclude_pmid_file = search_path + "pmids_to_excude.txt"
+    exclude_pmids = set()
+    with open(exclude_pmid_file, "r") as infile_fh:
+        for line in infile_fh:
```

Comment:
exclude_pmids = {line.rstrip().replace('PMID:', '') for line in infile_fh}

Reply:
fixed